<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>projectsDialog</class>
 <widget class="QDialog" name="projectsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Projects</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/podbicon.png</normaloff>:/podbicon.png</iconset>
  </property>
  <widget class="QGroupBox" name="projectsGroupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>521</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Projects</string>
   </property>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>501</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addRowPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Add a new project to the table</string>
    </property>
    <property name="text">
     <string>Add Project</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="savePushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>100</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="toolTip">
    <string>Save changes and close the dialog</string>
   </property>
   <property name="text">
    <string>Save Changes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelPushButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>250</y>
     <width>100</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="toolTip">
    <string>Close the dialog, discarding any changes</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
